### PR TITLE
administration: troubleshooting: added output params section

### PR DESCRIPTION
**Pull request by elenacbc**

### CODE REVIEW

File: administration/troubleshooting.md [hunk 1]
```diff
@@ -140,6 +140,50 @@ Now we should start seeing output similar to the following:
 [2022/10/21 10:49:25] [ info] [output:null:null.0] thread worker #0 stopping...
 [2022/10/21 10:49:25] [ info] [output:null:null.0] thread worker #0 stopped
 ```
+### Parameters for the output in Tap 
+When activating Tap, any plugin parameter can be given. These can be used to modify, for example, the output format, the name of the time key, the format of the date, etc.
+
+In the next example we will use the parameter ```"format": "json"``` to demonstrate how in Tap, stdout can be shown in Json format, however this could be sent to any output plugin.
```
Comment (lecaros) [inline administration/troubleshooting.md:146]: however... is redundant, we just explained that in the previous paragraph.

File: administration/troubleshooting.md [hunk 2]
```diff
@@ -140,6 +140,50 @@ Now we should start seeing output similar to the following:
...
+[0] dummy.0: [1674805636.973970215, {"message"=>"dummy"}]
+[{"date":1674805636.974008,"type":1,"trace_id":"2","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805636,"record":{"message":"dummy"}}],"start_time":1674805636,"end_time":1674805636},{"date":1674805636.974034,"type":3,"trace_id":"2","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805636,"record":{"message":"dummy"}}],"start_time":1674805636,"end_time":1674805636}]
+```
+This parameter shows stdout in Json format, however, as mentioned before, parameters can be passed to any plugin. 
+
+Please visit the following link for more information on other output parameters:
```
Comment (lecaros) [inline administration/troubleshooting.md:184]: output plugins, instead of output parameters.

**Review by lecaros — CHANGES_REQUESTED** [review]

File: administration/troubleshooting.md (hunk 2 above)
Comment (elenacbc) [reply administration/troubleshooting.md]: done

File: administration/troubleshooting.md (hunk 1 above)
Comment (elenacbc) [reply administration/troubleshooting.md]: done

**Pull request merged**